docs/source/reference/router/telemetry/trace-exporters/overview.mdx

```diff
@@ -108,6 +108,7 @@ telemetry:
 ```
 
 - `sampler` sets the sampling rate as a decimal percentage, `always_on`, or `always_off`. For example, setting `sampler: 0.1` samples 10% of your requests.
+  - [`always_off`](https://opentelemetry.io/docs/specs/otel/trace/tracestate-probability-sampling/#always-off-sampler) means that no spans will make it to your APM. It doesn't mean that sampling is disabled
```

Comment:
I might be missing something, but there doesn't seem to be a `#always-off-sampler` anchor on https://opentelemetry.io/docs/specs/otel/trace/tracestate-probability-sampling/#always-off-sampler.
I also wonder if adding a link to Otel's docs adds more to the explanation?

In any case, I'd recommend splitting this out into the three examples, will add a suggestion.